diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1bffffb..17f62fc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,13 +6,13 @@ enable_language(C CXX ASM)
 # BASIC SETUP
 #-------------------------------------------------------------------------------
 
-if(RISCV32)
-  message(STATUS "riscv32")
-  set(BITS 32)
-else()
-  message(STATUS "riscv64")
-  set(BITS 64)
-endif()
+#if(RISCV32)
+#  message(STATUS "riscv32")
+#  set(BITS 32)
+#else()
+#  message(STATUS "riscv64")
+set(BITS 64)
+#endif()
 
 set(prog_name    keystone-sdk)
 set(src_dir      ${CMAKE_CURRENT_LIST_DIR})
@@ -55,7 +55,7 @@ if (CMAKE_BUILD_TYPE STREQUAL "Debug")
     set(CMAKE_C_FLAGS ${CMAKE_C_FLAGS} -g)
 endif()
 
-use_riscv_toolchain(${BITS})
+#use_riscv_toolchain(${BITS})
 ################################################################################
 # BUILD PROJECTS
 ################################################################################
diff --git a/examples/CMakeLists.txt b/examples/CMakeLists.txt
index a70e6ca..85dd977 100644
--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -4,15 +4,15 @@ include(../macros.cmake)
 include(ExternalProject)
 find_package(Git REQUIRED)
 
-if(RISCV32)
-  message(STATUS "riscv32")
-  set(BITS 32)
-else()
-  message(STATUS "riscv64")
-  set(BITS 64)
-endif()
-
-use_riscv_toolchain(${BITS})
+#if(RISCV32)
+	#message(STATUS "riscv32")
+  #set(BITS 32)
+  #else()
+	# message(STATUS "riscv64")
+set(BITS 64)
+  #endif()
+
+#use_riscv_toolchain(${BITS})
 
 # check if SDK has been built and KEYSTONE_SDK_DIR environment variable exists
 if (NOT DEFINED ENV{KEYSTONE_SDK_DIR})
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 87aa61e..b684b0c 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,7 +1,7 @@
 include(${src_dir}/macros.cmake)
 
-set(libmodules              app edge host verifier)
-set(enabled_libmodules      app edge host verifier)
+set(libmodules              edge host verifier)
+set(enabled_libmodules      edge host verifier)
 
 # Common
 file(GLOB_RECURSE
